content/examples/02-next-pages/01-basics/01-generating-text.mdx

```diff
@@ -0,0 +1,72 @@
+---
+title: Generating Text
```

Comment:
for seo, wondering if "Generating Text (Next.js Pages Router)" might be better (also heading below)

Reply:
Changes here will reflect in the sidebar unfortunately, but I think we can append the framework by tweaking the metadata config in the layout, will dig deeper!